.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - 'v*' # older version branches
+    tags:
+      - '*'
+
+  pull_request: {}
+  schedule:
+  - cron:  '0 6 * * 0' # weekly, on sundays
+
+jobs:
+  test:
+    name: "Node ${{ matrix.node }} - ${{ matrix.os }}"
+    runs-on: "${{matrix.os}}-latest"
+
+    strategy:
+      matrix:
+        os: ['ubuntu', 'windows', 'macOS']
+        node: ['10', '12', '14']
```

Comment:
Doesn’t test node 16 (which is theoretically the point?)

Reply:
Yes you're right I should have added that. I'll split the CI stuff and then **add** 16 in this PR when it's rebased 👍